setup.ts

```diff
@@ -319,7 +319,8 @@ async function main(): Promise<void> {
   const { shouldSetMongoDb } = await inquirer.prompt({
     type: "confirm",
     name: "shouldSetMongoDb",
-    message: "Would you like to set up a MongoDB URL?",
+    message:
+      "(If using Docker dont set up a MongoDB URL)\nWould you like to set up a MongoDB URL?",
```

Comment:
The logic should ask whether this is a docker installation and if not, then the step should be skipped.

Reply:
Implemented the logic that ask whether docker installation and if yes, then the MongoURL step will be skipped